src/WebJobs.Script/Description/Node/Script/globalInitialization.js

```diff
@@ -8,6 +8,9 @@ return function (context, callback) {{
         context.handleUncaughtException(err.stack);
     }});
 
+    // TEMP HACK: workaround for https://github.com/tjanczuk/edge/issues/325
```

Comment:
Done.

---
In reply to: [70371964](https://github.com/Azure/azure-webjobs-sdk-script/pull/486#discussion_r70371964) [](ancestors = 70371964)